kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalSchemaRegistry.java

```diff
@@ -1,18 +1,38 @@
 package com.provectus.kafka.ui.model;
 
+import java.util.LinkedList;
 import java.util.List;
 import lombok.Builder;
 import lombok.Data;
 
 @Data
-@Builder(toBuilder = true)
 public class InternalSchemaRegistry {
   private final String username;
   private final String password;
-  private final List<String> url;
+  private final LinkedList<String> url;
```

Comment:
I guess that you split this into 2 methods because failoverlist doesn't implement List interface - you can use lombok @Delegate annotation. And you can leave just one field with type
```
  class FailoverUrlList {
    
    @Delegate
    private List<String> urls;
    //...
  }
  ```

Reply:
Actually I don't think you should be able to mutate urls once a FailoverUrlList has been created. What would be the point of being able to mutate it ? Or is the suffix List confusing for you because it does not implement the list interface ?

Reply:
it is not about mutations. You currently have 2 fields. url is for dto mappings and  urlList for failover. With @delegate you can merge it into single field: url with FailoverUrlList type

Reply:
I updated the PR according to your input.